src/tools/run-make-support/src/lib.rs

```diff
@@ -76,6 +76,20 @@ pub fn source_path() -> PathBuf {
     std::env::var("S").expect("S variable does not exist").into()
 }
 
+/// Creates a new symlink to a path on the filesystem, adjusting for Windows or Unix.
+#[cfg(target_os = "windows")]
+pub fn create_symlink<P: AsRef<Path>, Q: AsRef<Path>>(original: P, link: Q) {
+    use std::os::windows::fs;
+    fs::symlink_file(original, link).unwrap();
```

Comment:
Question: I don't know on top of my head if the panic if symlink fails give any context, would be nice to report "failed to create symlink `{link}` for `{original}`" in case of failure.